src/org/opensolaris/opengrok/configuration/messages/Message.java

```diff
@@ -108,6 +111,16 @@ public void setText(String text) {
         this.text = text;
     }
 
+    public void setTextFromFile(String filepath) {
+        try {
+            byte[] encoded = Files.readAllBytes(Paths.get(filepath));
+            this.text = new String(encoded);
+        } catch (IOException ex) {
+            System.err.println("Cannot read '" + filepath + "': " + ex);
+            System.exit(1);
```

Comment:
Do not use system.exit, this is not a main program. Throw an exception or return a boolean